build.sh

```diff
@@ -91,7 +90,9 @@ fi
 if hasArg --allgpuarch; then
     BUILD_ALL_GPU_ARCH=1
 fi
-if hasArg --multigpu; then
+if ahasArg --singlegpu; then
+    MULTIGPU=
+else
```

Comment:
For consistency, you can just make `MULTIGPU` have a new default of `--multigpu` on line ~51, then remove this `else` block.